src/components/fields/edit/LatLongEditor.jsx

```diff
@@ -24,14 +35,33 @@ export default function LatLongEditor({
   const [modalOpen, setModalOpen] = useState(false);
   const [mapLatLng, setMapLatLng] = useState(null);
 
+  const {
+    latitudeString: initialLatitudeString,
+    longitudeString: initialLongitudeString,
+  } = useMemo(() => deriveGpsStringsFromValue(value));
+
+  const [currentLatitudeString, setCurrentLatitudeString] = useState(
+    initialLatitudeString,
+  );
+  const [
+    currentLongitudeString,
+    setCurrentLongitudeString,
+  ] = useState(initialLongitudeString);
+
+  useEffect(
+    () => {
+      const {
+        latitudeString,
+        longitudeString,
+      } = deriveGpsStringsFromValue(value);
```

Comment:
I don't understand why the useMemo above is necessary if this derive method gets run every time the values change anyway.

Reply:
well the useMemo prevents it from happening twice. since this useMemo has no dependencies it will only get run once on the component's first render